### PR TITLE
Adding what is next section in Cheaha account management page

### LINKED ISSUE #701: Add a brief next steps section on the Cheaha Account Management page advertising some of our more common requests and useful services
### What would you like to see added?

Add a section "Next Steps" or "What can I do with my account?" at: https://docs.rc.uab.edu/account_management/cheaha_account/

The section might have something like what follows

## What can I do with my account?

Research Computing offers services to cover a wide range of researcher needs, for students, staff and faculty, and for both labs and research cores.

We're always happy to provide support for your Research Computing needs, you need only [contact us] (https://docs.rc.uab.edu/#how-to-contact-us)

### For Students, Staff, and Faculty

- Get Started with Open OnDemand (https://docs.rc.uab.edu/cheaha/open_ondemand/)
- Additional Learning Resources (https://docs.rc.uab.edu/education/training_resources/)
- Data Science Journal Club Course (https://docs.rc.uab.edu/education/courses/#data-science-journal-club-course)

### For Lab PIs and Core Directors

- No-cost storage offerings (https://docs.rc.uab.edu/data_management/storage/#what-type-of-storage-do-i-need)
    - GPFS: Hot storage, compute adjacent, directly accessible from Cheaha (https://docs.rc.uab.edu/data_management/storage/#what-shared-storage-solutions-are-available)
    - LTS: Cool storage, large capacity (https://docs.rc.uab.edu/data_management/lts/)
- Batch computing (https://docs.rc.uab.edu/cheaha/slurm/introduction/)
    - Desktop, Jupyter, RStudio, Matlab, and more (https://docs.rc.uab.edu/cheaha/open_ondemand/)
    - GPUs (https://docs.rc.uab.edu/cheaha/slurm/gpu/)
- On-prem cloud computing (https://docs.rc.uab.edu/uab_cloud/)
    - Tutorial (https://docs.rc.uab.edu/uab_cloud/tutorial/)
    - Web servers (https://docs.rc.uab.edu/uab_cloud/remote_access/#make-instances-publically-accessible-from-the-internet)
- Not seeing what you need? Contact our team of facilitators! (https://docs.rc.uab.edu/#how-to-contact-us)

### PR DESCRIPTION
**Pull request by bdu-birhanu**
# Pull Request

## Overview
The aim of this pull request is to add "a What can I do with my account" section to the Cheaha account management page

## Proposed Changes
- added "a What can I do with my account" on Cheaha account management page
- list and crosslink to some of our more common requests and useful services

## Related Issues
Fixes #701

### CODE REVIEW

**Comment by bdu-birhanu** [comment]
Since the two issues ( #701  and #471  ) are addressed in separate branches, now, this pull request is awaiting on issue #471 instead of #432

File: docs/account_management/cheaha_account.md [hunk 1]
```diff
@@ -78,11 +82,37 @@ Periodically, we review all researcher accounts to ensure they are authorized to
...
+
+### For Students, Staff, and Faculty
+
+- [Get Started with Open OnDemand](../cheaha/open_ondemand/index.md)
+- [Additional Learning Resources](../education/training_resources.md)
+- [Data Science Journal Club Course](../education/courses.md#data-science-journal-club-course)
+
+### For Lab PIs and Core Directors
+
+- [No-cost storage offerings](../data_management/storage.md#what-type-of-storage-do-i-need)
```
Comment (wwarriner) [inline docs/account_management/cheaha_account.md:99]: Add one more to the storage offerings, with a link.

- Transfer data with Globus

File: docs/account_management/cheaha_account.md [hunk 2]
```diff
@@ -78,11 +82,37 @@ Periodically, we review all researcher accounts to ensure they are authorized to
...
+    - [GPFS](../data_management/storage.md#what-shared-storage-solutions-are-available): Hot storage, compute adjacent, directly accessible from Cheaha
+    - [LTS](../data_management/lts/index.md): Cool storage, large capacity
+- [Batch computing](../cheaha/slurm/introduction.md)
+    - [Desktop](../cheaha/open_ondemand/hpc_desktop.md), [Jupyter](../cheaha/open_ondemand/ood_jupyter.md), [RStudio](../cheaha/open_ondemand/ood_rstudio.md), [Matlab](../cheaha/open_ondemand/ood_matlab.md), and more
+    - [GPUs](../cheaha/slurm/gpu.md)
+- [On-prem cloud computing](../uab_cloud/index.md)
+    - [Tutorial](../uab_cloud/tutorial/index.md)
+    - [Web servers](../uab_cloud/remote_access.md#make-instances-publically-accessible-from-the-internet)
+
+  If you are unable to find what you need, please contact our team [here](../index.md#how-to-contact-us).
```
Comment (wwarriner) [inline docs/account_management/cheaha_account.md:109]: Please don't use "here" as a link. Use "Contact Us" or "Contact Support". You may need to rephrase the sentence some.

File: docs/account_management/cheaha_account.md [hunk 3]
```diff
@@ -78,11 +82,37 @@ Periodically, we review all researcher accounts to ensure they are authorized to
...
+
 ## Cheaha Account and Group Membership FAQ
 
 Our Cheaha system is robust, but errors may occur due to general platform connectivity issues or missing components. Below are FAQs for self-service Cheaha account creation and a troubleshooting guide for common issues:
 
-- **Which credentials should I use?** Please visit [how do I login to research computing services](../account_management/index.md/#how-do-i-login-to-research-computing-services).
+- **Which credentials should I use?** Please visit [how do I login to research computing services](../account_management/index.md#how-do-i-login-to-research-computing-services).
```
Comment (wwarriner) [inline docs/account_management/cheaha_account.md:115]: Capitalize links to specific sections. So use How Do I Login to Research Computing Services.

**Review by wwarriner — CHANGES_REQUESTED** [review]
Looks good, needs a couple small fixes and good to merge.

File: docs/account_management/cheaha_account.md (hunk 2 above)
Comment (bdu-birhanu) [reply docs/account_management/cheaha_account.md]: Fixed. Thank you.

**Review by wwarriner — APPROVED** [review]

**Pull request merged**